vmf/scripts/mods/vmf/modules/core/packages.lua

```diff
@@ -0,0 +1,135 @@
+local vmf = get_mod("VMF")
+
+local _initialized = false
+
+local _mod_handles = {}
+local _queued_packages = {}
+local _loading_package = nil
+local _loaded_packages = {}
+
+function VMFMod:load_package(package_name, callback)
```

Comment:
This (and the other) functions have no argument type-checking. Ie,

```suggestion
function VMFMod:load_package(package_name, callback)
  if vmf.check_wrong_argument_type(self, "load_package", "package_name", package_name, "string") or
     vmf.check_wrong_argument_type(self, "load_package", "callback", callback, "function", "nil)
  then
    return
  end
```

Can you include these checks to keep the codebase consistent?